fe2-android/app/src/test/java/com/github/dedis/popstellar/utility/handler/ChirpHandlerTest.java

```diff
@@ -40,6 +43,7 @@
 
 import io.reactivex.Completable;
 
+@HiltAndroidTest
```

Comment:
You do not need to use Hilt there. You can just create the serverRepository through its constructor.
`new ServerRepository()`